packages/storybook/src/nimble/table-column/mapping/table-column-mapping.mdx

```diff
@@ -28,40 +34,57 @@ string values as an icon, a spinner, text, or an icon/spinner with text in the
 <Controls of={tableColumnMappingStories.mappingColumn} />
 <ComponentApisLink />
 
-### Icon Mapping
+### Icon mapping
 
 <Description of={iconMappingStories} />
 <Controls of={iconMappingStories.iconMapping} />
 
-### Spinner Mapping
+### Spinner mapping
 
 <Description of={spinnerMappingStories} />
 <Controls of={spinnerMappingStories.spinnerMapping} />
 
-### Text Mapping
+### Text mapping
 
 <Description of={textMappingStories} />
 <Controls of={textMappingStories.textMapping} />
 
-### Empty Mapping
+### Empty mapping
 
 <Description of={emptyMappingStories} />
 <Controls of={emptyMappingStories.emptyMapping} />
 
 ## Usage
 
-### Best Practices
-
--   Provide a mapping for every expected record value. Because grouping is
-    performed on the record value, non-mapped record values can result in
-    multiple groups without group labels.
-    -   To improve grouping behavior of a value that shouldn't be rendered in
-        table cells, use a <Tag name={mappingEmptyTag}/> element to provide text
-        for that value's group row.
--   Avoid having multiple values that map to the same text because grouping and
-    sorting is done on the record value rather than the mapped text.
-
-### Blazor Usage
+### Best practices
+
+- Provide a mapping for every expected record value. Because grouping is
+  performed on the record value, non-mapped record values can result in
+  multiple groups without group labels.
+  - To improve grouping behavior of a value that shouldn't be rendered in
+    table cells, use a <Tag name={mappingEmptyTag}/> element to provide text
+    for that value's group row.
+- Avoid having multiple values that map to the same text because grouping and
+  sorting is done on the record value rather than the mapped text.
```

Comment:
Avoid having multiple values that map to the same text because the text is used for grouping and would result in multiple groups with the same text label.